### ISSUE TITLE
confused about the scene_box/scene_scale and pose

### ISSUE THREAD

**Issue by zjcs** [nerfstudio-project/nerfstudio#2380]
**Is your feature request related to a problem? Please describe.**

The input pose of NeRF always is generated by COLMAP without scale, so the poses can be scaled and normailized in [-1, 1]. While in read application, the image pose has scale, and I am confused how to implement the scale process in nerfstudio.

I notice that there are two processes related to the pose: scene_box and scene_scale.
1. scene_box: initialize the SceneBox(torch.tensor([[-1/poses.abs().max()]*3, [1/poses.abs().max()]*3])
2. scene_scale: (1) poses[:,:3,3]*=scene_scale, (2) dataparser_scale=scene_scale.

Back to my confusion, I has thress questions: 
(1) how does these three variable works?
(2) when my input poses has real scale, how should I process the poses/scene_box/scene_box?
(3) Especially, when my loss is in a real scale, what should I be careful of?

**Describe the solution you'd like**
1. A supplement readme.md about how they works and some code examples.

**Describe alternatives you've considered**
NO

**Additional context**
NO
